.rubocop.yml

```diff
@@ -79,3 +79,29 @@ Style/ParenthesesAroundCondition:
 
 Style/PreferredHashMethods:
   Enabled: false
+
+Minitest/AssertEmptyLiteral:
+  Enabled: true
+
+Minitest/AssertNil:
+  Exclude:
+    - 'test/graphql/queries/personal_access_token_query_test.rb'
+  Enabled: true
+
+Minitest/RefuteNil:
+  Enabled: true
+
+Layout/ArgumentAlignment:
+  EnforcedStyle: with_fixed_indentation
+
+Layout/SpaceAroundOperators:
+  Enabled: true
+
+Layout/EmptyLinesAroundAttributeAccessor:
+  Enabled: true
+
+Layout/EmptyLineBetweenDefs:
+  Enabled: true
+
+Layout/FirstHashElementIndentation:
```

Comment:
Looking at the examples in the definition of this rule I don't have a strong opinion here (I'm not a fan of any of those), but looking at the changes in `Fix rubocop Layout/FirstHashElementIndentation` commit, I'd like to also move other key-values to be aligned with the first key (https://github.com/theforeman/foreman/pull/9977/commits/2f8031c19856008cbd45d76b9e09fcd5d1d57cb9#diff-6ad3cd4f298f82f58bf997e42ac904fcb45a3f583653e84e2831225582a96f03L39-L43).